src/main/src/main/java/org/geoserver/security/VectorAccessLimits.java

```diff
@@ -166,6 +183,22 @@ private void writeProperties(List<PropertyName> attributes, ObjectOutputStream o
         }
     }
 
+    public Geometry getClipVectorFilter() {
+        return clipVectorFilter;
+    }
+
+    public void setClipVectorFilter(Geometry clipVectorFilter) {
+        this.clipVectorFilter = clipVectorFilter;
+    }
+
+    public Geometry getIntersectVectorFilter() {
```

Comment:
Actually, since there is read filter, wondering if GeoFence cannot just build the full read filter by itself, rather than adding this field. This would also simplify SecuredFeatureSource.